bootstrap/configs.go

```diff
@@ -82,23 +82,23 @@ type ConfigRepository interface {
 	// to indicate operation failure.
 	Update(ctx context.Context, cfg Config) error
 
-	// UpdateCerts updates and returns an existing Config certificate and owner.
+	// UpdateCerts updates and returns an existing Config certificate and domainID.
 	// A non-nil error is returned to indicate operation failure.
-	UpdateCert(ctx context.Context, owner, thingID, clientCert, clientKey, caCert string) (Config, error)
+	UpdateCert(ctx context.Context, domain_id, thingID, clientCert, clientKey, caCert string) (Config, error)
```

Comment:
use camelcase for go naming. Have a look at https://go.dev/doc/effective_go

```suggestion
	UpdateCert(ctx context.Context, domainID, thingID, clientCert, clientKey, caCert string) (Config, error)
```